### PR TITLE
Task2branch

### LINKED ISSUE #2: Task 2
### Goal:
Create a CLI that given a `user_id`, can print a table of items in the order that associates with the `user_id`, create a new order, add new items to and edit existing order, and set the ready state of the current user.
 - [x] Create supermarket database:
Store price and name of the items available
  - Example fields:
    - `item_id`
    - `name`
    - `price`
  - Hard code test data *(modify the database file directly through your editor)*
  - Create a query method that given a `item_id`, retrieve its name and price 
 - [x] Print function:
Print columns of the items' property and the order's items as row, a prompt asking for user input and listing out available commands.
  - Can decide whether the list is printed for every command or only when explicitly told to
  - Optionally print the `order_id` as the title of the table
  - Only print items create by the user and items whose publicity is set to true
  - Retrieve the order's data of the `user_id`from the **order database** *(by Task1, return test data for now)*
  - Retrieve the items' data of the `item_id`*s* from the **supermarket database**
  - Join the data from the 2 databases to print all the columns of the table
 - [x] Create new order function:
After the user selected this command, return if a new order is created or any error *(by Task1, return test data for now)*.
 - [x] Add new item & Edit item function:
  - These are meant to be 2 separated functions
  - Create an insert method that adds the new item to the order's `items` in the **order database** *(refer to the database structure created by Task1)*
  - Create an update method that edits the element in the order's `items` specified by the user in the **order database** 
  - Calling either of the 2 functions will reset the ready state *(e.g. isReady)* of of all users in the order with the same `order_id` in the **order database** and set the reset state *(e.g. isReset)* to true if the operation involves a public order. Create an update method to do so
  - Return if the insertion/update is successful or any error
  - Notify the users about the change of reset all isReady status.
 - [x] Set ready state function:
Create an update method that changes the ready state *(e.g. isReady)* of the current user in the **order database**

### PR DESCRIPTION
**Pull request by YidingWang05**
All tests have been passed. Ready for the next step.

### CODE REVIEW

File: databases/order.json
Comment (ghost) [inline databases/order.json:1]: Please remove all the test data from this file.

File: src/orderSetUp.py
Comment (ghost) [inline src/orderSetUp.py:1]: Remove this file if it is for testing purposes only.

File: src/test.py
Comment (ghost) [inline src/test.py:1]: Remove this file if it is for testing purposes only.

File: .DS_Store
Comment (ghost) [inline .DS_Store:1]: Remove this file.

File: src/shopping.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+current_dir = os.path.dirname(__file__)
+supermarket_db_path = os.path.join(
+    current_dir, "../databases/supermarket.json"
+)
+order_db_path = os.path.join(current_dir, "../databases/order.json")
+
+supermarket = TinyDB(supermarket_db_path)
+order_db = TinyDB(order_db_path)
+Item = Query()
```
Comment (ghost) [inline src/shopping.py:13]: Unify `Item = Query()` and `Product = Query()` into a single Query() object, and replace all previous instance of `Item` and `Product`. 
E.g. :
<pre>
QUERY = Query()

def get_order(id):
    return order_db.get(QUERY.order_id == id)
</pre>

File: src/shopping.py [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+    print("\nPersonal Items: ")
+    print(
+        tabulate(get_personal_table(items, user_id), headers, tablefmt="grid")
+    )
+
+    return True
+    # return public_table, personal_table, headers, headers_public
+
+
+def create_new_order_status(isCreated, order_id):
```
Comment (ghost) [inline src/shopping.py]: Comment this function out or maybe simply delete it.

**Review by ghost — CHANGES_REQUESTED** [review]
Please update according to the comments

File: databases/supermarket.json
Comment (ghost) [inline databases/supermarket.json:1]: This one is actually needed.

**Review by ghost — CHANGES_REQUESTED** [review]

File: databases/supermarket.json
Comment (ghost) [reply databases/supermarket.json:1]: Just copy the content from previous commits.

Comment (YidingWang05) [reply databases/supermarket.json:1]: Ok, no problems

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**